compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -67,11 +68,9 @@ static llvm::cl::list<int> mmt4dVectorSizes(
     "iree-codegen-llvm-mmt4d-vector-size",
     llvm::cl::desc("linalg.mmt4d vector tile size"), llvm::cl::ZeroOrMore);
 
-static llvm::cl::opt<int> defaultWorkgroupTileSize(
-    "iree-codegen-llvm-generic-ops-workgroup-size",
-    llvm::cl::desc(
-        "linalg.generic and linalg.indexed_generic workgroup tile size"),
-    llvm::cl::init(64));
+static llvm::cl::opt<int> defaultDistTileSize(
```

Comment:
Nit: `defaultDistTilSizes` -> `defaultDistributionTileSizes` (just being consistent with above).

Reply:
I'm a bit confused... Aren't distribution and flow level tile sizes the ones that were equivalent, same for workgroup and parallel/reduction tile sizes? Why are we replacing workgroup with distribution?

I think the replacement should be s/flow/distribution and s/workgroup/vector/. Then we will have vector parallel, vector reduction, cache parallel, cache reduction... At least I have followed that convention in multi-level tiling. WDYT?

Reply:
Some workgroup term was mis-used in this file, and I replaced them with the correct one -- which is distribution. There were two flags about tile sizes. One is `*workgroup*` and the other is `*vector*`. I think the former one should be replaced with `distribution`, does it make sense?